### ISSUE TITLE
Workspace directories should not change by navigating between folders

### ISSUE THREAD

**Issue by MariusAlbrecht** [flxzt/rnote#1014]
**Describe the bug**  
When navigating inside a workspace, that is double clicking a directory or using the "Move dir up" button the actual root directory of a workspace changes.

**To Reproduce**  
1. create workspace at location `<somepath>/x/`
2. navigate to `<somepath>/x/y/` or `<somepath>/`
3. click "Edit selected workspace"
4. see changed workspace location

**Expected behavior**  
The root directory should stay the same, only the _displayed_ directory should change. In particular should the same root dir be displayed at every startup regardless of navigation in earlier sessions.
The root dir should be changeable (only) by using the "Edit [selected] workspace" menu.

**Additional context**  
n/a

**Comment by Doublonmousse** [comment]
Should a "reset" button be added to reset these workspace to their original folder ? 

Seeing that the directory changed from the original workspace folder should be visible too (maybe a dot notation to show changes from the initial change)

I feel like some of the workspace design has to change to accomodate this change. At the moment the current behavior matches the design and stopping at just changing when the workspace directory changes would be confusing without these two extra pointers to understand the new behavior

**Comment by MariusAlbrecht** [comment]
that'd certainly be a possibility.
For me personally things would work as expected if everything stayed as is now (especially current navigation and ui) except for the fact that workspaces lose their "navigation state" when the app is closed.
This means upon startup it'd be at the root location again.
I can absolutely see a case for a more explicit solution, too. My version works for my intuition, and I can't really guess it'd work for others as well

**Comment by Doublonmousse** [comment]
Yeah on second thought the dot thing is probably not a good idea.

Just a single button to reset the view back to the root folder of the workspace is probably more than enough to convey the new behavior and make it easier to reset things back to the initial state of the workspace directory if needed

**Comment by KelvinNovais** [comment]
I agree with the author, it doesn't make sense to have a workspace root directory that keeps moving.

Some people pointed about a "reset button", that in my opinion is the workspace button itself: whenever you click it, you go to the workspace you have set.

**Comment by Doublonmousse** [comment]
Oh that's a neat solution. 

Well I'm all for it then (use the workspace button to reset the view).
I don't have any disagreement left with that proposed solution.
